python/cudf/cudf/core/frame.py

```diff
@@ -2310,31 +2309,33 @@ def drop_duplicates(
         result._postprocess_columns(self)
         return result
 
-    def replace(self, to_replace, replacement):
+    def replace(self, to_replace: Any, replacement: Any) -> "Frame":
```

Comment:
Just FYI, if you do:

```
from __future__ import annotations
```

then you don't have to use a string to refer to `Frame` in the type annotations.  I believe that's the preferred convention.

I know there's plenty of other functions here that use `"Frame"`. We can fix them all in another PR; but just letting you know about this convention for the benefit of future PRs :)

Reply:
Changed this just in this file I made edits. 👍